logstash-core/src/test/java/org/logstash/common/io/DeadLetterQueueReaderTest.java

```diff
@@ -134,9 +134,63 @@ public void testSeek() throws Exception {
         assertThat(entry.getReason(), equalTo("543"));
     }
 
+
     @Test
-    public void testInvalidDirectory()  throws Exception {
-        DeadLetterQueueReader reader = new DeadLetterQueueReader(dir);
-        assertThat(reader.pollEntry(100), is(nullValue()));
+    public void testWriteStopSmallWriteSeekByTimestamp() throws Exception {
+        DeadLetterQueueWriter writeManager = new DeadLetterQueueWriter(dir, 10000000, 10000000);
```

Comment:
@robbavey this thing is a `Closeable` now => we should be able to `try-with-resources` here :)